### DISCUSSION TITLE
Option to disable Calendar tree when making a new note

### DISCUSSION THREAD

**Discussion by Stitch10925** [TriliumNext/Trilium#6024] [Ideas]
### Describe feature

When creating a new note Trillium has the habit of creating a whole calendar tree (Calendar > Year > Day > Note).

It would be nice if this could be disabled and that the note, of no parent item is selected, is simply added to the root.

Now I always have to delete the calendar structure since I do not wish to use it.

**Comment by meichthys** [comment]
This can be configured using the `#datePattern`, `#monthPattern`, `#yearPattern`, and `#calendarRoot` attributes. 
See here for more details: https://triliumnext.github.io/Docs/Wiki/day-notes.html

It seems like you would want your root note to have the `#calendarRoot` attribute

**Comment by Stitch10925** [comment]
Thank you for the fast reply, however, I am not sure how this will stop the creation of the whole calendar structure. Won't this just put the calendar under my root note?

**Comment by eliandoran** [comment]
@Stitch10925 , you can add `#inbox` on the root note (or any other note for that matter) and it will put them directly in there, without any calendar structure.
